handlers/slog/options.go

```diff
@@ -0,0 +1,50 @@
+package sloghandler
+
+import (
+	"io"
+	"log/slog"
+
+	"github.com/vpakhuchyi/censor"
+)
+
+// Option represents a set of options for configuring the Slog Handler along with a censor processor.
```

Comment:
It feels like the description isn't correct. Based on the implementation. Option is just an option, not a set (or a few of them).
I mean, each option may apply only the only characteristic/action.

Reply:
Make sense, thank you